src/app/profile/settings/notifications/toggle-bar.component.less

```diff
@@ -0,0 +1,66 @@
+@import (reference) '../../../../assets/stylesheets/shared/osio.less';
+
+.toggle {
+  display: none;
+  // add default box-sizing for this scope
+  &,
+  &:after,
+  &:before,
+  & *,
+  & *:after,
+  & *:before,
+  & + .toggle-btn {
+    box-sizing: border-box;
+    &::selection {
+      background: none;
+    }
+  }
+  + .toggle-btn {
+    outline: 0;
+    display: block;
+    width: 4em;
+    height: 2em;
+    position: relative;
+    cursor: pointer;
+    user-select: none;
+    &:after,
+    &:before {
+      position: relative;
+      display: block;
+      content: '';
+      width: 50%;
+      height: 100%;
+    }
+    &:after {
+      left: 0;
+    }
+    &:before {
+      display: none;
+    }
+  }
+  // &:checked + .toggle-btn:after {
+  //   left: 50%;
+  // }
+}
+
+.toggle-light {
+  + .toggle-btn {
+    background: @color-pf-black-200;
+    border-radius: 1px;
+    border: 1px solid #bbb;
+    padding: 0;
+    //transition: all .4s ease;
+    &:after {
+      background: @color-pf-blue-400;
+      box-shadow: 0 0 2px rgba(3, 3, 3, .4);
+      background-image: linear-gradient(to bottom, @color-pf-black-100  0, @color-pf-black-200  100%);
+      background-repeat: repeat-x;
+      //transition: all .3s ease;
```

Comment:
Remove commented out code.

Reply:
I did this on the presumption that we would eventually want transitions to be activated, at which point we would un-comment. Is this undesirable ?

Reply:
It's best to remove the unused code, even if it'll be implemented at a later date. We can standardize transitions, and then just use a variable for easier updates.